runners/genlayer-py-std/src/genlayer/gl/_internal/runner.py

```
@@ -15,7 +15,7 @@
 from genlayer.py.types import Rollback
 import genlayer.py._internal.reflect as reflect
 
-import genlayer.std._internal.gl_call as gl_call
+import genlayer.gl._internal.gl_call as gl_call
```

Pylint (3.3.7)

[refactor] Use 'from genlayer.gl._internal import gl_call' instead

(R0402)